src/core/public/chrome/ui/header/nav_link.tsx

```diff
@@ -176,7 +120,7 @@ export function createRecentNavLink(
 
   return {
     href,
-    label: truncateRecentItemLabel(label),
```

Comment:
Where was the `truncateRecentItemLabel` logic moved to?

Reply:
I deleted it 💣 

I confirmed it wasn't working and removing it didn't change anything for the new nav or the legacy nav. 

Also, it was only ever for the legacy nav so I didn't think debugging+fixing what it was supposed to do was worth it.